lineman/app/services/form_service.coffee

```diff
@@ -13,23 +13,49 @@ angular.module('loomioApp').factory 'FormService', ($rootScope, FlashService, $f
       422: 'unprocessableEntity'
       500: 'internalServerError'
 
+    prepare = (scope, model, options) ->
+      FlashService.loading()
+      scope.isDisabled = true
+      model.setErrors
```

Comment:
also I added a clearErrors() method because it makes more sense when reading stuff like this.